src/CardContainer/CardContainer.js

```diff
@@ -23,6 +24,6 @@ const CardContainer = ({movieData}) => {
 export default CardContainer;
 
 CardContainer.propTypes = {
-  movieData: PropTypes.object,
+  movieData: PropTypes.array,
```

Comment:
Guess what, if you put in the wrong proptype, you're going to give yourself an error.